raiden_contracts/utils/sign_utils.py

```diff
@@ -3,6 +3,8 @@
 
 
 def sign(privkey: str, msg: bytes, v=0) -> bytes:
+    if isinstance(privkey, bytes):
```

Comment:
hmm shouldn't that be an assertion? The type hint expects a `str`. So it's a programming error to provide bytes for the `privkey`